akka-javasdk-maven/akka-javasdk-parent/pom.xml

```diff
@@ -284,7 +278,28 @@
                         </systemPropertyVariables>
                     </configuration>
                 </plugin>
-
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-failsafe-plugin</artifactId>
+                    <version>${maven-failsafe-plugin.version}</version>
+                    <executions>
+                        <execution>
+                            <goals>
+                                <goal>integration-test</goal>
+                                <goal>verify</goal>
```

Comment:
`mvn integration-test`, it just works!

```suggestion
                                <goal>integration-test</goal>
```

Reply:
yup, double checked and it just works. The integration tests are this way run by surefire but works nevertheless. Deleting code is one of my favorite hobbies :)